pkg/products/threescale/reconciler.go

```diff
@@ -433,6 +439,13 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	err = r.addResourceThreshold(ctx, serverClient)
+	r.log.Infof("ensureDeploymentConfigsReadyForEnvoy-SidecarContainer", l.Fields{"phase": phase})
+	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
```

Comment:
not sure we need to care about the `phase` here :thinking: Since `addResourceThreshold` isn't updating it

Reply:
how would you suggest i write this instead?

Reply:
hmm, just take it away? :thinking: 
```
	err = r.addResourceThreshold(ctx, serverClient)
	r.log.Info("ensureDeploymentConfigsReadyForEnvoy-SidecarContainer")
	if err != nil {
		events.HandleError(r.recorder, installation, phase, "Failed to ensure deployment configs for envoy-sidecar were ready", err)
		return phase, err
	}
```